### ISSUE TITLE
Nice to have: Last Lap/Interval stats

### ISSUE THREAD

**Issue by pace6666** [jonasoreland/runnerup#36]
Allows to show last lap/interval (Time/Distance/Pace) during activity to compare last lap/interval with current lap/interval.  Maybe an option to hide/show in settings.

**Comment by gerhardol** [comment]
It would be nice to see the lap (interval) history when exercising (for instance during rest laps).

**Comment by davidedelvento** [comment]
Yes, this is something on my plate (unless @cmmata beats me with his GUI work). See https://github.com/davidedelvento/runnerup/issues/25
